### PR TITLE
Log network open

### LINKED ISSUE #545: Add an INFO log line that the network is open

### PR DESCRIPTION
**Pull request by achamayou**
fix #545

### CODE REVIEW

File: src/node/nodestate.h [hunk 1]
```diff
@@ -1067,6 +1067,7 @@ namespace ccf
       active_service->status = ServiceStatus::OPEN;
       service_view->put(0, active_service.value());
 
+      LOG_INFO_FMT("Changing network status from OPENING to OPEN");
```
Comment (jumaffre) [inline src/node/nodestate.h]: Should this be in the global hook for the service table instead? So that this is displayed on all nodes.

**Comment by cimetrics** [comment]
log_network_open@[3759 aka 20200109.7](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3759&view=results) vs master ewma over 20 builds from [3644](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3644&view=results) to [3750](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3750&view=results)
images

File: src/node/nodestate.h (hunk 1 above)
Comment (achamayou) [reply src/node/nodestate.h]: And it's displayed when it's globally committed rather than tentatively executed on the leader? Yes!

Comment (jumaffre) [reply src/node/nodestate.h]: Yes. There's already a log message there but it should be more made explicit.

Comment (achamayou) [reply src/node/nodestate.h]: Which it turns we were already doing, albeit not with the same wording.

**Comment by cimetrics** [comment]
log_network_open@[3769 aka 20200109.12](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3769&view=results) vs master ewma over 20 builds from [3656](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3656&view=results) to [3767](https://msrc-ccf.visualstudio.com/CCF/_build/results?buildId=3767&view=results)
images

**Pull request merged**